packages/components/molecules/f-media-element/src/components/MediaElement.vue

```diff
@@ -1,18 +1,120 @@
 <template>
     <div
-        :class="$style['c-mediaElement']"
-        data-test-id="mediaElement-component">
-        {{ componentName }}
+        data-test-id="mediaElement-component"
+        :class="[$style['c-mediaElement'], { [$style['c-mediaElement--stack']]: stacked }, { [$style['c-mediaElement--reverse']]: reverse }]">
+        <div :class="[$style['c-mediaElement-content'], contentAlignClass]">
+            <h3 :class="[$style['c-mediaElement-title'], $style['c-mediaElement-text'], textSizeClass]">
```

Comment:
Also, should this have `c-mediaElement-title` and `c-mediaElement-text` on it? I'd just have `c-mediaElement-title` and then use selectors where it makes sense to share the styles between the two classes.

Reply:
I have now replaced this and have added a font size class that is applied to both